index.js

```diff
@@ -56,7 +56,8 @@ function top (opts) {
         time: us,
         percent: (cpuDelta.system + cpuDelta.user) / us,
         system: cpuDelta.system,
-        user: cpuDelta.user
+        user: cpuDelta.user,
+        count: os.cpus().length
```

Comment:
just use the cpus variable defined above, os.cpus() is not safe.

Reply:
Oh I did not know that, I picked it up in the first place becase the doc said that it may differ from OSes.

Reply:
Cc https://docs.libuv.org/en/v1.x/misc.html#c.uv_available_parallelism

Reply:
They said "estimate" also ...

Reply:
But the Node doc said: 

> os.cpus().length should not be used to calculate the amount of parallelism available to an application. Use [os.availableParallelism()](https://nodejs.org/api/os.html#osavailableparallelism) for this purpose

Reply:
> cpus() will fail on android for permission reasons etc

thanks for sharing :)

Reply:
i learned that last week lol